pkg/daemon/ceph/osd/volume.go

```diff
@@ -524,23 +524,36 @@ func (a *OsdAgent) useRawMode(context *clusterd.Context, pvcBacked bool) (bool,
 		useRawMode = false
```

Comment:
based on docs, encryptedDevice can also be set per device.Config

https://rook.io/docs/rook/v1.6/ceph-cluster-crd.html#osd-configuration-settings